tests/utils.py

```diff
@@ -1328,3 +1330,37 @@ def master_seed(seed=1234, set_random=True, set_numpy=True, set_tensorflow=False
                 torch.cuda.manual_seed_all(seed)
         except ImportError:
             logger.info("Could not set random seed for PyTorch.")
+
+
+def get_gan_inverse_gan_ft():
+    import tensorflow as tf
+    from models.create_inverse_gan_models import build_gan_graph, build_inverse_gan_graph
+
+    if tf.__version__[0] == "2":
+        return None, None, None
```

Comment:
Should we raise `NotImplementedError` with an explanation instead of returning `None`s?

Reply:
this is part of a larger discussion about the ART tests. At the moment I followed the same convention we're using for classifiers, which is to return None whenever no classifier meets the set of required combination of specifications (framework specific, defended, not defended etc.). In my opinion, yes I think we should be raising a `NotImplementedError`. The reason why this approach was taken for the classifiers is because within the existing test code, a lot of combinations of requirements were not been tested. So, now that the pytests provide the ability to go through each possible combination, raising this error would make a lot of tests fail. At the moment, the tests check whether there is a classifier returned or not, if there is it performs the tests as it was before using pytest if there's none it just ignores it. I think if we decided to more to raising an error, then we would have to catch that error within the test itself. I'm happy to do that for the Gans if you want? I just wanted to stay consistent with the rest of the tests but if we decide to move to raising Error then I can go with that approach here.

Reply:
Ok, I see. In this case let's leave as is and we could maybe pick this up again in the work for moving all classifier tests to pytest.